spec/game_spec.rb

```diff
@@ -0,0 +1,25 @@
+require_relative 'spec_helper'
+
+describe Game do
+  context 'Testing Games' do
+    before(:each) do
+      @game = Game.new('2011-06-12', 'Y', '2014-05-12')
+    end
+
+    it 'Should be an instance of the Game class' do
+      expect(@game).to be_instance_of Game
+    end
+
+    it 'Should return true as the game is a multiplayer' do
+      expect(@game.multiplayer).to eq 'Y'
```

Comment:
In line 13 you said `Should return true as the game is a multiplayer` but on line 14 you're expecting the value to be equal to `Y`